### PR TITLE
Com.voxdsp.voxel paint

### PR DESCRIPTION
**Pull request by BeverlyCode**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: .gitmodules
Comment (JakobDev) [inline .gitmodules:1]: Why did you include this submodule? It is not used in your Manifest.

**Comment by hfiguiere** [comment]
this is because submission MUST be made against the `new-pr` branch.

**Comment by omnivoxel** [comment]
> this is because submission MUST be made against the `new-pr` branch.

yes my mistake i must have branched from master, shall I re-submit a new branch?

**Comment by JakobDev** [comment]
You now that you can create a text build local?

**Comment by JakobDev** [comment]
Sorry, this was a typo. I mean test build.

File: .gitmodules
Comment (hfiguiere) [inline .gitmodules:1]: the submodule should be removed as it is not used.

File: com.voxdsp.VoxelPaint.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    build-commands:
+      - cc main.c -lSDL2 -lGLESv2 -lEGL -Ofast -lm -o voxelpaint
+      - install -Dm 0755 voxelpaint -t ${FLATPAK_DEST}/bin
+      - install -Dm 0644 flatpak/voxelpaint.desktop ${FLATPAK_DEST}/share/applications/voxelpaint.desktop
+      - install -Dm 0644 flatpak/voxelpaint.appdata.xml ${FLATPAK_DEST}/share/metainfo/voxelpaint.appdata.xml
+      - install -Dm 0644 flatpak/voxelpaint.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/voxelpaint.png
+    sources:
+      - type: git
+        url: https://github.com/mrbid/voxelpaint.git
+        commit: 1601430
```
Comment (hfiguiere) [inline com.voxdsp.VoxelPaint.yaml:26]: please use long form commits sha.

**Comment by barthalion** [comment]
Can you answer the questions in the PR template?

**Comment by razzeee** [comment]
My guess is, that https://github.com/flathub/website/blob/main/backend/app/worker.py#L74 computed somehow before your app summary got a `timestamp` and therefor computed as a `null` which then turned into 1970 and sorted it back to the beginning. We only ever write this once.

But it's hard to validate, without access to the redis database.

**Pull request closed without merging**